### PR TITLE
UI

### PR DESCRIPTION
**Pull request by osama-maree**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,104 +0,0 @@
...
-  Temp = [],
-  inpEdit = "";
-
-function fetchData() {
-  Todos = JSON.parse(localStorage.getItem("todos"));
-  Temp = Todos;
-  ShowTodo();
-}
-fetchData();
-function ShowTodo() {
```
Comment (m5atib) [inline main.js]: please write the functions name in `camelCase`

File: main.js [hunk 2]
```diff
@@ -1,104 +0,0 @@
-let Attributes = document.getElementsByTagName("tbody"),
```
Comment (m5atib) [inline main.js]: please write the variable names in `camelCase`

File: main.js [hunk 3]
```diff
@@ -1,104 +0,0 @@
...
-  count = document.getElementsByTagName("p"),
-  input = document.getElementById("input"),
-  addBtn = document.getElementById("addBtn"),
-  Search = document.getElementById("search"),
```
Comment (m5atib) [inline main.js]: please write the variable names in `camelCase`

File: main.js [hunk 4]
```diff
@@ -1,104 +0,0 @@
...
-  Todos,
-  Temp = [],
```
Comment (m5atib) [inline main.js]: please write the variable names in camelCase

File: main.js (hunk 1 above)
Comment (m5atib) [inline main.js]: Please rearrange the functions. so this `ShowTodo` function should be before `fetchData` function

File: main.js [hunk 5]
```diff
@@ -1,104 +0,0 @@
...
-    rows += `<td>${element.completed}</td>`;
-    rows += `<td>  <button class="custom-button mb" onclick="DeleteTodo(${element.id})">Delete</button>
-                   <button class="custom-button" style="background-color: #4CAF50;"onclick="DoneTodo(${element.id})">Done</button>
-               </td>`;
-    rows += "</tr>";
-  });
-  Attributes[0].innerHTML = rows;
-  count[0].textContent = `Total tasks: ${Todos.length}`;
-}
-function DeleteTodo(id) {
```
Comment (m5atib) [inline main.js]: please write the function name in camelCase

File: main.js [hunk 6]
```diff
@@ -1,104 +0,0 @@
...
-function ToggleEdit(id) {
-  Todos.forEach((todo) => {
-    if (todo.id === id && !todo.completed) {
-      todo.edit = !todo.edit;
-    }
-  });
-  localStorage.setItem("todos", JSON.stringify(Todos));
-  fetchData();
-}
-function SaveChange(val, id) {
```
Comment (m5atib) [inline main.js]: please write the function name in camelCase

we usually write the classes only in `PascalCase `or the Functional Component in react

**Review by m5atib — COMMENTED** [review]
Please review the needed changes

File: main.js (hunk 1 above)
Comment (osama-maree) [reply main.js]: Okay, I'll edit it

Comment (osama-maree) [reply main.js]: But it works fine Is there something wrong?

**Pull request merged**